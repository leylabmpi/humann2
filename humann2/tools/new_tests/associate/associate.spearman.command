humann2_associate \
		  -i associate.input \
		  -o associate.spearman.output \
		  -L CATE \
		  -m CONT \
		  -t continuous \
		  -f 0.2 \