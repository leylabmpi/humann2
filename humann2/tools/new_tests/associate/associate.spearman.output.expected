# FEATURE	EFFECT_SIZE	P-VALUE	Q-VALUE
F009	0.7455	0.01333	0.1857
F017	0.7212	0.01857	0.1857
